codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java

```diff
@@ -99,7 +99,7 @@ public void addRuntimeConfigValues(
                 .map(ServiceTrait::getArnNamespace)
                 .orElse(null);
 
-        if (signingName != null) {
+        if (signingName != null && target.equals(LanguageTarget.SHARED)) {
```

Comment:
This target check should be a wrapper for the current `if/else`, or there will be log lines for other targets.

Reply:
Fixed in 694083cbd.  Moved target check up to wrap entire `signingName` section.